app/cli/cmd/root.go

```diff
@@ -97,7 +95,7 @@ func Execute(l zerolog.Logger) error {
 
 func NewRootCmd(l zerolog.Logger) *cobra.Command {
 	rootCmd := &cobra.Command{
-		Use:           appName,
+		Use:           "chainloop",
```

Comment:
how come? what changed?

Reply:
You are right, that shouldn't change - it is because I moved the name to the common package and I got rid of the change lately. The change is no longer here.